.travis.yml

```diff
@@ -11,9 +11,17 @@ cache:
     - node_modules
 branches:
   only: [master]
-jobs:
+matrix:
   include:
     - stage: Tests
```

Comment:
Do we need this line `stage: Tests`? I was comparing this with `.travis.yml` in ZAF [here](https://github.com/zendesk/zendesk_app_framework/blob/master/.travis.yml)

Reply:
No, I don't think we do!